Source/WebDriver/Session.h

```diff
@@ -236,6 +255,12 @@ class Session : public RefCounted<Session> {
     std::optional<String> m_currentParentBrowsingContext;
     HashMap<String, InputSource> m_activeInputSources;
     HashMap<String, InputSourceState> m_inputStateTable;
+#if ENABLE(WEBDRIVER_BIDI)
+    bool m_bidiFlag;
```

Comment:
What does m_bidiFlag actually signify? I think a better name here would be helpful in understanding the code. https://webkit.org/code-style-guidelines/#names-bool

Also, the capitalization is inconsistent between bidiFlag and setBiDiFlag.
https://webkit.org/code-style-guidelines/#names-basic

Reply:
Renamed to `hasBiDiEnabled`, to mean that the given `Session` has a related BiDi channel attached.